amr-wind/CMakeLists.txt

```diff
@@ -37,8 +37,11 @@ if(AMR_WIND_ENABLE_MASA)
 endif()
 
 # Generate AMReX build information file
+include(amr-wind-utils)
```

Comment:
I thought that the `include` from top-level would be inherited here... is that not the case?

Reply:
No, I tried, but it didn't seem to work. I will double check again though. I expected it to.

Reply:
Something must have been in flux that made this not work initially because it does now that I tried it again.